src/backend/marsha/core/tests/api/video/test_create.py

```diff
@@ -78,6 +78,7 @@ def test_api_video_create_by_playlist_admin(self):
                 "lti_id": "video_one",
                 "playlist": str(playlist.id),
                 "title": "Some video",
+                "license": "CC_BY-ND",
```

Comment:
```suggestion
                "license": "IV",
```

Reply:
https://youtu.be/n-OaEJ4oNAQ?feature=shared